css-view-transitions-2/Overview.bs

```diff
@@ -389,12 +396,15 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 ## Monkey patches to HTML ## {#monkey-patch-to-html}
 
 	<div algorithm="monkey patch to apply the history step">
-		Prepend a step at the beginning of the task [=queue a global task|queued=] on |navigable|'s [=active window=]
+		Prepend these steps at the beginning of the task [=queue a global task|queued=] on |navigable|'s [=active window=]
 		when <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#apply-the-history-step">applying the history step</a> (14.11.1, <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#updating-the-traversable:queue-a-global-task-3">here</a>):
 
-		This monkey-patch step assumes a boolean |changingNavigationContinuation|, a [=/navigable=] |navigable|, a {{Document}} |oldDocument|, a {{Document}} |newDocument|, and a {{NavigationType}} |navigationType|.
+		This monkey-patch step assumes a boolean |changingNavigationContinuation|, a [=/navigable=] |navigable|, a {{Document}} |oldDocument|, a {{Document}} |newDocument|, a {{NavigationType}} |navigationType|,
+		and a <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#user-navigation-involvement">user navigation involvement</a> |userInvolvementForNavigateEvents|.
```

Comment:
Can we export "user navigation involvement" from the html spec? These steps will eventually move to the html spec but we'll still have the term referenced in the navigation descriptor definition above.

Reply:
Done: https://github.com/whatwg/html/pull/10058. Will update this spec once that export is merged.